deploy.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+UPDATE=$1
+DEST_DIR=$2
+FED_KEY=""
+MAIN_BLOCK_NUMBER=""
+SIDE_BLOCK_NUMBER=""
+
+cd $DEST_DIR/federator
```

Comment:
Please check that both `UPDATE` and `DEST_DIR` aren't empty, and also quote the `cd` parameter.

Reply:
Done